samples/exposed-migration/src/main/kotlin/Application.kt

```diff
@@ -0,0 +1,62 @@
+import org.flywaydb.core.Flyway
+import org.jetbrains.exposed.sql.Database
+import org.jetbrains.exposed.sql.ExperimentalDatabaseMigrationApi
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.jetbrains.exposed.sql.insert
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.jetbrains.exposed.sql.vendors.currentDialect
+import java.util.*
+
+const val URL = "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1"
+const val USER = "root"
+const val PASSWORD = ""
+
+@OptIn(ExperimentalDatabaseMigrationApi::class)
+fun main() {
+    val database = Database.connect(
+        url = URL,
+        user = USER,
+        driver = "org.h2.Driver",
+        password = PASSWORD
+    )
+
+    val migrationsDirectory = "migrations" // Location of migration scripts
+
+    val flyway = Flyway.configure()
+        .dataSource(URL, USER, PASSWORD)
+        .locations("filesystem:$migrationsDirectory")
+        .baselineOnMigrate(true) // Used when migrating an existing database for the first time
+        .load()
+
+    transaction(database) {
+        SchemaUtils.create(UsersV1)
+
+        UsersV1.insert {
+            it[id] = UUID.randomUUID()
+            it[email] = "root1@root.com"
+        }
+
+        println("*** Before migration ***")
+        println("Primary key: ${currentDialect.existingPrimaryKeys(UsersV1)[UsersV1]}")
+    }
+
+    transaction {
+        MigrationUtils.generateMigrationScript(
+            UsersV2,
+            scriptDirectory = migrationsDirectory,
+            scriptName = "V2__Add_primary_key",
+        )
+
+        flyway.migrate()
+    }
```

Comment:
Maybe a comment could be useful here reminding of the full path where new script is expected to be found.

Or a comment that line 50 could be commented out if the migration script wants to be reviewed first (to tie into what was said in the README).